packages/algob/src/lib/tx.ts

```diff
@@ -191,14 +191,15 @@ export async function transferMicroAlgosLsig (
   fromAccount: Account,
   toAccountAddr: string,
   amountMicroAlgos: number,
-  lsig: LogicSig): Promise<tx.ConfirmedTxInfo> {
-  const params = await deployer.algodClient.getTransactionParams().do();
+  lsig: LogicSig,
+  payFlags: TxParams): Promise<tx.ConfirmedTxInfo> {
+  const params = await mkSuggestedParams(deployer.algodClient, payFlags);
 
   const receiver = toAccountAddr;
-  const note = tx.encodeObj("ALGO PAID");
+  const note = tx.encodeObj(payFlags.note);
```

Comment:
Since note is an optional param, what happens when note is not passed?

Reply:
yes, this needs to be changed